airflow/cli/commands/scheduler_command.py

```diff
@@ -34,12 +35,17 @@
 from airflow.utils.cli import process_subdir, setup_locations, setup_logging, sigint_handler, sigquit_handler
 from airflow.utils.scheduler_health import serve_health_check
 
+log = logging.getLogger(__name__)
+
 
 def _run_scheduler_job(job_runner: SchedulerJobRunner, *, skip_serve_logs: bool) -> None:
     InternalApiConfig.force_database_direct_access()
     enable_health_check = conf.getboolean("scheduler", "ENABLE_HEALTH_CHECK")
     with _serve_logs(skip_serve_logs), _serve_health_check(enable_health_check):
-        run_job(job=job_runner.job, execute_callable=job_runner._execute)
+        try:
+            run_job(job=job_runner.job, execute_callable=job_runner._execute)
+        except Exception:
+            log.exception("Exception when running scheduler job")
```

Comment:
Should we also log what the exception message was?

Reply:
https://docs.python.org/3/library/logging.html#logging.exception
Calling `log.exception` will automatically include the exception information so there is no need to include it separately.

Reply:
yes, sorry missed reading the logging level.